examples/aws-ml-app-demo/serving.py

```diff
@@ -10,9 +12,21 @@ def predict(input_json):
     print(f'input_json={input_json}')
     input_dict = json.loads(input_json)
     model, version = _MODEL_STORE.load_latest_model()
+    if version is None:
+        return jsonify(
```

Comment:
Maybe you can build the json [like here](https://github.com/apache/incubator-liminal/blob/master/tests/runners/airflow/build/http/python/test_python_server_image_builder.py#L88):
**json_string = '{"key1": "val1", "key2": "val2"}'**

[Then](https://github.com/apache/incubator-liminal/blob/master/tests/runners/airflow/build/http/python/test_python_server_image_builder.py#L99): json.loads(json_string)

Instead of an extra importing.